met/src/basic/vx_cal/is_leap_year.cc

```diff
@@ -91,13 +92,16 @@ void increase_one_month(int &year, int &month) {
 unixtime add_to_unixtime(unixtime base_unixtime,
     int sec_per_unit, double time_value, bool no_leap) {
   unixtime ut;
-  bool debug = false;
+  const char *ptr = getenv("MET_DEBUG");
+  bool debug = (NULL != ptr && 0 < strlen(ptr));
```

Comment:
```suggestion
  const char *ptr = getenv("MET_DEBUG");
  bool debug = (NULL != ptr && 0 < strlen(ptr));
```

Replace that with this
```
bool debug = mlog.verbosity_level() >= 5;
```
Or substitute in whatever level value you'd like.
I think that's preferable to introducing a new MET_DEBUG environment variable.